home-assistant.code-workspace

```diff
@@ -0,0 +1,7 @@
+{
+	"folders": [
```

Comment:
Please remove the stray files.

Reply:
Done

Reply:
(Now done for real... sorry bout that, having to get used to different git tools / dev flows)